src/components/Components/Labels/Badge.js

```diff
@@ -68,6 +97,10 @@ const AutoCompleteBadgeStyled = styled.div`
   background-color: ${props => getBadgeColor(props)};
 `;
 
+const StatusBadgeStyled = styled(AutoCompleteBadgeStyled)`
+  background-color: ${status => getStatusBadgeColor(status)};
```

Comment:
Just one little thing-- the function inside the template literal here takes `props` as an argument, which you've called `status`. Probably this should be:
` ${props => getStatusBadgeColor(props.status)}`
and in the function definition:
```
const getStatusBadgeColor = status => {
  ...
  switch (status) {
      ...
```